app/helpers/ordinal_columns_helper.rb

```diff
@@ -0,0 +1,32 @@
+module OrdinalColumnsHelper
+  def ordinal_column_class(columns)
+    case columns
+    when 1
+      ''
+    when 2
+      'width-one-half'
+    when 3
+      'width-one-third'
+    when 4
+      'width-one-fourth'
+    when 5
+      'width-one-fifth'
+    when 6
+      'width-one-sixth'
+    when 7
+      'width-one-seventh'
+    when 8
+      'width-one-eigth'
```

Comment:
Does this need to be `eighth`?

Reply:
it does, yes. Will fix. I'm decent at programming, less good at spelling